pkg/profiler/cpu/bpf/maps/maps.go

```diff
@@ -102,7 +102,7 @@ const (
 			mapping_t mappings[MAX_MAPPINGS_PER_PROCESS];
 		} process_info_t;
 	*/
-	mappingInfoSizeBytes = 8 + 8 + (maxMappingsPerProcess * 8 * 5)
+	mappingInfoSizeBytes = 8*4 + (maxMappingsPerProcess * 8 * 5)
```

Comment:
I don't think I understand this change, `(maxMappingsPerProcess * 8 * 5)` makes sense because it's the mapping struct (each field is 8 bytes), but where do the static `8 * 4` come from?

Reply:
four extra fields  8 bytes each here https://github.com/parca-dev/parca-agent/blob/7a532053d10a220dc32362481f82487c80e55665/bpf/unwinders/native.bpf.c#L205-L208

see how in go comment above it is 
```
typedef struct {
	u64 is_jit_compiler;
	u64 len;
	mapping_t mappings[MAX_MAPPINGS_PER_PROCESS];
} process_info_t;
```

and in kernel it is 

```
typedef struct {
  u64 should_use_fp_by_default;
  u64 is_jit_compiler;
  u64 interpreter_type;
  u64 len;
  mapping_t mappings[MAX_MAPPINGS_PER_PROCESS];
} process_info_t;
```

so it used to be `8 + 8 + (maxMappingsPerProcess * 8 * 5)`
and now it should be `8 + 8 + 8 + 8 + (maxMappingsPerProcess * 8 * 5)`

Does it make sense?

Reply:
Can you update the comment with that?

Reply:
updated